platform/solr/solr-factory/src/main/java/org/codice/solr/factory/impl/HttpSolrClientFactory.java

```diff
@@ -62,6 +67,11 @@
  */
 @Deprecated
 public final class HttpSolrClientFactory implements SolrClientFactory {
+
+  public static final String[] DEFAULT_PROTOCOLS;
```

Comment:
✏️ By keeping these publics and making them arrays, you are not opened for the elements to be changed by anybody that accesses them. 

❓ Should we removed those completely from this API in favor of retrieving them directly from the system properties everywhere we need them? This will help with the fact that this class will be part of an implementation bundle and not exposed as a service whatsoever once @tbatie finishes the current ticket we have.

Reply:
I like that. I had that in my refactored branch. I'll add it into this one.

Reply:
I remember why I didn't keep that refactoring. The HttpSolrWrapper is a class that really should not exist in the first place. It duplicates a lot what happens in HttpSolrClientFactory. Still it would be good to remove the dependency.